presto-main/src/main/java/com/facebook/presto/metadata/FunctionManager.java

```diff
@@ -102,6 +105,13 @@
     private final Map<CatalogSchemaPrefix, String> functionNamespaces = new ConcurrentHashMap<>();
     private final Map<String, FunctionNamespaceManager<?>> functionNamespaceManagers = new ConcurrentHashMap<>();
 
+    private final CounterStat exactMatchFunctionCount = new CounterStat();
```

Comment:
Maybe just use `AtomicLong` for these stats. We can do differential on these numbers on minute granularity to get the per-min count. The reason to avoid using `CounterStat` on frequently called methods is due to its lock. `CounterStat` has heavy lock contention.